src/http-api/prefix.ts

```diff
@@ -16,11 +16,7 @@ limitations under the License.
 
 export enum ClientPrefix {
     /**
-     * A constant representing the URI path for release 0 of the Client-Server HTTP API.
-     */
-    R0 = "/_matrix/client/r0",
-    /**
-     * A constant representing the URI path for the legacy release v1 of the Client-Server HTTP API.
+     A constant representing the URI path for Client-Server API endpoints versioned at v1.
```

Comment:
```suggestion
     * A constant representing the URI path for Client-Server API endpoints versioned at v1.
```

Reply:
Argh, need to figure out why WebStorm strips the leading `*` when copy-pasting into a jsdoc block, thanks